numba/tests/test_np_functions.py

```diff
@@ -2448,6 +2452,56 @@ def a_variations():
 
         self.assertIn("Cannot np.flip on UniTuple", str(raises.exception))
 
+    def test_logspace_basic(self):
+        pyfunc = logspace
+        cfunc = jit(nopython=True)(pyfunc)
+
+        self.assertPreciseEqual(pyfunc(1, 60), cfunc(1, 60))
+        self.assertPreciseEqual(pyfunc(-60, 1), cfunc(-60, 1))
+        self.assertPreciseEqual(pyfunc(-1, -60), cfunc(-1, -60))
+
+        self.assertPreciseEqual(pyfunc(1.0, 60.0), cfunc(1.0, 60.0))
+        self.assertPreciseEqual(pyfunc(-60.0, -1.0), cfunc(-60.0, -1.0))
+        self.assertPreciseEqual(pyfunc(-1.0, 60.0), cfunc(-1.0, 60.0))
+
+        self.assertPreciseEqual(pyfunc(0.0, np.e), cfunc(0.0, np.e))
+        self.assertPreciseEqual(pyfunc(0.0, np.pi), cfunc(0.0, np.pi))
```

Comment:
Consider writing this as a loop over input tuples? It'd also be good to add in some variation in the type of the input, e.g. complex values.